verilog/tools/kythe/README.md

```diff
@@ -12,12 +12,22 @@ cross references for easy navigation, and even name completion in editors.
 ## Usage
 
 ```
-verible-verilog-kythe-extractor [options] <files...>
+verible-verilog-kythe-extractor [options] --file_list_path FILE
 
-    --output_path (File path where to write the extracted Kythe facts in JSON
-      format.); default: "";
     --printextraction (Whether or not to print the extracted general indexing
-      facts from the middle layer)); default: false;
-    --print_kythe_facts_json (Whether or not to print the extracted kythe facts);
-      default: false;
+      facts tree from the middle layer)); default: false;
+    --print_kythe_facts (Determines how to print Kythe indexing facts.
+                        Options:
+                            json: Outputs Kythe facts in JSON format.
+                            proto: Outputs Kythe facts in proto format);
+                        default: json;
+    --file_list_path (The path to the file list which contains the names of SystemVerilog files.
+                      The files should be ordered by definition dependencies)
+    --file_list_root (The absolute location which we prepend to the files in the file list (where listed files are relative to);
+                      default: the place of invocation.
```

Comment:
shouldn't this change to the directory of the file list.

Reply:
That would assume that filelists live in projects' root directories, which is not always the case.  Current default (. = $PWD) is ok.